### ISSUE TITLE
Storage Queue Ouput Binding - Set visibilityTimeout?

### ISSUE THREAD

**Issue by stephen776** [Azure/azure-functions-dotnet-worker#493]
Is there any way to set the queue message `visibilityTimeout` when returning a `QueueOutput` as an output binding?

**Comment by anthonychu** [comment]
I don't believe this is possible now without changes to the binding extension. https://github.com/Azure/azure-sdk-for-net/issues/21887
